src/modules/Dashboard/containers/Dashboard/index.tsx

```diff
@@ -1,16 +1,32 @@
 import React from 'react';
 
-import { Container, Header } from './styles';
+import { useAuth } from '../../../Auth/contexts/auth';
+import { Body, Container, Header } from './styles';
 
 const Dashboard: React.FC = () => {
+  const { profile, hasRole } = useAuth();
+
+  const all_roles: Array<string> = ['admin', 'teacher', 'student'];
+  const valid_roles: Array<string> = all_roles.filter(hasRole);
+
   return (
     <Container>
       <Header>
         <div>
           <h1>Dashboard</h1>
-          <span>Welcome back, Ricardo!</span>
+          <span>Welcome back, {profile?.firstName}!</span>
```

Comment:
We should use translations, but we can open an issue to do this

Reply:
Yes, we should. But you know, this PR address integration with a third-party authentication server (Keycloak), not all application issues we have right now.

Reply:
Yep, I know that.
Just saying to open an issue so we can solve this another time.